src/index.ts

```diff
@@ -13,8 +13,13 @@ import createImpl, {
 export * from './vanilla'
 
 // For server-side rendering: https://github.com/react-spring/zustand/pull/34
-const useIsoLayoutEffect =
-  typeof window === 'undefined' ? useEffect : useLayoutEffect
+// Deno support: https://github.com/pmndrs/zustand/issues/347
+const isSSR =
+  typeof window === 'undefined' ||
+  !window.navigator ||
+  /ServerSideRendering/.test(window.navigator.userAgent)
```

Comment:
```suggestion
```

Reply:
The point is this is not only for deno. This is useful for something else like jsdom.